src/examples/src/widgets/wizard/Basic.tsx

```diff
@@ -0,0 +1,95 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Example from '../../Example';
+import Wizard, { Step, StepItem } from '@dojo/widgets/wizard';
+import Button from '@dojo/widgets/button';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache }).properties();
+
+export default factory(function Basic({ middleware: { icache } }) {
+	let activeStep = icache.getOrSet('activeStep', 1);
+	let steps = icache.getOrSet<StepItem[]>('steps', [
+		{ status: 'complete' },
+		{ status: 'inProgress' },
+		{ status: 'pending' }
+	]);
+	return (
+		<Example>
+			<Wizard
+				steps={steps}
+				onStep={(stepIndex) => {
+					icache.set('activeStep', stepIndex);
+					icache.set(
+						'steps',
+						steps.map((_, index) => {
```

Comment:
won't this overwrite any existing `errror` states?

Reply:
Yes, but there aren't any in this basic example

Reply:
fair enough, just thinking about the best thing to be showing users is all, it's not a bit issue

Reply:
Fair enough, I can add an if to return `error` if there is one. No harm in doing so